docs/blog/2022-05-02-arch-linux-installation-guide.mdx

```diff
@@ -0,0 +1,355 @@
+Below is the command to encrypt that partition using LUKS.
+
+    cryptsetup --cipher aes-exts-plain64 --hash sha256 --use-random --verify-passphrase luksFormat --type luks2 /dev/nvme0n1p2
+
+Now we'll open the encrypted partition with:
+
+    cryptsetup open /dev/nvme0n1p2 luks
+
+:::note
+("luks" is just a placeholder, you can use a name of your choice, but remember to continue with your naming convention if changed):
+:::
+
+## File System Creation
+Format the EFI partition with FAT32 and give it the label EFI:
+
+    mkfs.vfat -F32 -n EFI /dev/nvme0n1p1
+
+Format the root partition with BTRFS and give it the label ROOT.
+
+    mkfs.btrfs -L ROOT /dev/mapper/luks
+
+:::note
+If you didn’t open the LUKS container under the name “luks” you must adjust the command accordingly
+:::
+
+## Create and Mount Sub Volumes
+
+Create sub volumes for root, home, the package cache, snapshots and the entire BTRFS file system:
+
+    mount /dev/mapper/luks /mnt
+    btrfs sub create /mnt/@
+    btrfs sub create /mnt/@swap
+    btrfs sub create /mnt/@home
+    btrfs sub create /mnt/@pkg
+    btrfs sub create /mnt/@snapshots
+    umount /mnt
+
+:::note
+If you don’t want to use a swap file, it is not required to create and mount the sub volume @swap.
+:::
+
+Mount the sub volumes:
+
+    mount -o noatime,nodiratime,compress=zstd,space_cache=v2,ssd,subvol=@ /dev/mapper/luks /mnt
+    mkdir -p /mnt/{boot,home,var/cache/pacman/pkg,.snapshots,btrfs}
+    mount -o noatime,nodiratime,compress=zstd,space_cache=v2,ssd,subvol=@home /dev/mapper/luks /mnt/home
+    mount -o noatime,nodiratime,compress=zstd,space_cache=v2,ssd,subvol=@pkg /dev/mapper/luks /mnt/var/cache/pacman/pkg
+    mount -o noatime,nodiratime,compress=zstd,space_cache=v2,ssd,subvol=@snapshots /dev/mapper/luks /mnt/.snapshots
+    mount -o noatime,nodiratime,compress=zstd,space_cache=v2,ssd,subvolid=5 /dev/mapper/luks /mnt/btrfs
+
+Mount the EFI partition
+
+    mount /dev/nvme0n1p1 /mnt/boot
+
+:::note
+Optional - The following is for those wanting to use a swapfile
+:::
+
+Create swap file (taken from Btrfs - Swap file and Swap file):
+
+    cd /mnt/btrfs/@swap
+    truncate -s 0 ./swapfile
+    chattr +C ./swapfile
+    btrfs property set ./swapfile compression none
+    dd if=/dev/zero of=./swapfile bs=1M count=<FILE-SIZE-IN-MiB> status=progress
+    chmod 600 ./swapfile
+    mkswap ./swapfile
+    swapon ./swapfile
+    cd -
+
+## Base System and /etc/fstab
+We will be installing Arch Linux with the packages we need to get started.
+For our machine we will be using [intel-ucode](https://wiki.archlinux.org/title/microcode) as the X1 has an intel CPU.setup
+
+:::note
+If your machine has an amd cpu, please replace **intel-ucode** with [amd-ucode](https://archlinux.org/packages/?name=amd-ucode).
+If you're looking for another text editor or any other package you want to install with Arch here is the place to do so, just add it to the command below.
+:::
+
```

Comment:
This is a good place to add that ANY other packages they want can go here.  If they know they need vim, just throw in vim! Itll save time later